Add CML - Cisco Modeling Labs (#131)
networktocode/awesome-network-automation: README.md

# Training

> - [CML - Training videos @ learningnetwork.cisco.com](https://learningnetwork.cisco.com/s/cml-training-videos)

Reviewer: Can you add a sentence here.

Author: Oops, not sure what happened there 😄 Fixed now!

# Network Emulators

>  - [CML](https://www.cisco.com/c/en/us/products/cloud-systems-management/modeling-labs/index.html) - Cisco Modeling Labs (CML), which replaces VIRL. CML is a network emulator that has a easy to use gui frontend, but also offers a comprehensive API.

Reviewer: ```suggestion
 - [CML](https://www.cisco.com/c/en/us/products/cloud-systems-management/modeling-labs/index.html) - Cisco Modeling Labs (CML), which replaces VIRL, is a network emulator with both an API and GUI frontend.
```

I try to keep the opinions if not an official tag line to a minimum.
